pkg/agent/manager/private.go

```diff
@@ -38,6 +38,8 @@ type manager struct {
 	bundleCachePath string
 
 	subscribers subscribers
+
+	agentConnPool []*grpc.ClientConn
```

Comment:
might be useful for this to be a map, keyed by spiffe id of the client cert in use? not sure how you would identify which connection to use otherwise

Reply:
Yeah, I had to do that :)